### PR TITLE
Add restart messages, bot and checkboxes

### PR DESCRIPTION
**Pull request by vargasjona**
This PR has the following changes:

Add forwardRef to RasaChat so it can handle inner functions from parent components
Add resetMessages method to clear messages
Add resetBot method to reset bot (Rasa bot will need a /reset intent which clean conversation using action)
Add checkbox

I hope to create a sample code soon, also please update the license to MIT

https://user-images.githubusercontent.com/23468720/149640935-1b0cbffc-4e92-46b6-aa48-cdfa6efdeb42.mp4

n

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,8 +108,13 @@ export default StyleSheet.create({
 ### TODO List:
 
 - [ ] Reset bot on destroy
+<<<<<<< HEAD
```
Comment (hungvu193) [inline README.md:111]: seem like you need to resolve conflict here

**Comment by hungvu193** [comment]
Can you provide an example about checkbox feature? I think you should squash your commits into  1 commit it will look better.
Also if you can provide checkbox feature example in Readme it would be awesome 👍 . You're doing such a great job, thank you so much @jonra1993 💯

**Comment by vargasjona** [comment]
> Can you provide an example about checkbox feature? I think you should squash your commits into 1 commit it will look better. Also if you can provide checkbox feature example in Readme it would be awesome 👍 . You're doing such a great job, thank you so much @jonra1993 💯

Sure I am going to have an example soon

**Comment by hungvu193** [comment]
> > Can you provide an example about checkbox feature? I think you should squash your commits into 1 commit it will look better. Also if you can provide checkbox feature example in Readme it would be awesome 👍 . You're doing such a great job, thank you so much @jonra1993 💯
> 
> Sure I am going to have an example soon

awesome, I left a small review, please fix it, squash your commits and it's all good

**Comment by vargasjona** [comment]
> > > Can you provide an example about checkbox feature? I think you should squash your commits into 1 commit it will look better. Also if you can provide checkbox feature example in Readme it would be awesome 👍 . You're doing such a great job, thank you so much @jonra1993 💯
> > 
> > 
> > Sure I am going to have an example soon
> 
> awesome, I left a small review, please fix it, squash your commits and it's all good

**Comment by vargasjona** [comment]
> > > Can you provide an example about checkbox feature? I think you should squash your commits into 1 commit it will look better. Also if you can provide checkbox feature example in Readme it would be awesome 👍 . You're doing such a great job, thank you so much @jonra1993 💯
> > 
> > 
> > Sure I am going to have an example soon
> 
> awesome, I left a small review, please fix it, squash your commits and it's all good

Hello, @hungvu193 Thanks for the feedback. I am going to solve it and work on sample on the weekend

**Pull request closed without merging**